docs/spec/v1beta2/buckets.md

```diff
@@ -504,6 +505,39 @@ spec:
   endpoint: https://testfluxsas.blob.core.windows.net
 ```
 
+##### Azure Blob SAS Token example
+
+```yaml
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: Bucket
+metadata:
+  name: azure-sas-token
+  namespace: default
+spec:
+  interval: 5m0s
+  provider: azure
+  bucketName: <bucket-name>
+  endpoint: https://<account-name>.blob.core.windows.net
+  secretRef:
+    name: azure-key
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azure-key
+  namespace: default
+type: Opaque
+data:
+  sasKey: <base64>
+```
+
+The query values from the `sasKey` data field in the Secrets gets merged with the `spec.endpoint` of the `Bucket`.
+If the same key is present in the both of them, the token takes precedence.
```

Comment:
I feel this could use more explanation, in combination with looking at the doc page of the added ref. I had at least the following questions:

- Does the `sasKey` only contain the "SAS token"?
- If so, is it expected to be prefixed with `?` as [suggested in the documentation](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview#sas-token)?
- What does

  > If the same key is present in the both of them, the token takes precedence.

  entail precisely?